WowPacketParser/Enums/TrophyLockCode.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace WowPacketParser.Enums
+{
+    public enum TrophyLockCode
+    {
+        CodeWrongAchievement = 34,
```

Comment:
remove "Code" term in CodeWrongAchievement and CodeSuccess. This is not C++, enums here kinda work the same way as `enum class` in C++, that means the enum name itself is required - e.g. TrophyLockCode.WrongAchievement.